cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizationBootstrapper.java

```diff
@@ -1,109 +0,0 @@
-/*
- * Copyright © 2016 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package co.cask.cdap.security.authorization;
-
-import co.cask.cdap.common.conf.CConfiguration;
-import co.cask.cdap.common.conf.Constants;
-import co.cask.cdap.proto.id.InstanceId;
-import co.cask.cdap.proto.id.NamespaceId;
-import co.cask.cdap.proto.security.Action;
-import co.cask.cdap.proto.security.Principal;
-import co.cask.cdap.security.spi.authorization.PrivilegesManager;
-import com.google.common.base.Splitter;
-import com.google.common.base.Throwables;
-import com.google.inject.Inject;
-import org.apache.hadoop.security.UserGroupInformation;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.io.IOException;
-import java.util.Collections;
-import java.util.EnumSet;
-import java.util.HashSet;
-import java.util.Set;
-
-/**
- * A class to bootstrap authorization
- */
-public class AuthorizationBootstrapper {
-
-  private static final Logger LOG = LoggerFactory.getLogger(AuthorizationBootstrapper.class);
-
-  private final boolean enabled;
-  private final PrivilegesManager privilegesManager;
-  private final Principal systemUser;
-  private final Set<Principal> adminUsers;
-  private final InstanceId instanceId;
-
-  @Inject
-  AuthorizationBootstrapper(CConfiguration cConf, PrivilegesManager privilegesManager) {
-    this.enabled =
-      cConf.getBoolean(Constants.Security.ENABLED) && cConf.getBoolean(Constants.Security.Authorization.ENABLED);
-    String currentUser;
-    try {
-      currentUser = UserGroupInformation.getCurrentUser().getShortUserName();
-    } catch (IOException e) {
-      throw Throwables.propagate(e);
-    }
-    this.systemUser = new Principal(currentUser, Principal.PrincipalType.USER);
-    this.adminUsers = getAdminUsers(cConf);
-    if (enabled && adminUsers.isEmpty()) {
-      LOG.info("Admin users specified by {} is empty.", Constants.Security.Authorization.ADMIN_USERS);
```

Comment:
Do we need to remove this constant from cdap-site? I dont think it is useful any more